lib/api/core/models/security/role.js

```diff
@@ -1,12 +1,17 @@
-var
+'use strict';
+
+const
   _ = require('lodash'),
   Promise = require('bluebird'),
   async = require('async'),
   vm = require('vm'),
   BadRequestError = require('kuzzle-common-objects').errors.BadRequestError,
   InternalError = require('kuzzle-common-objects').errors.InternalError,
   ParseError = require('kuzzle-common-objects').errors.ParseError,
-  Request = require('kuzzle-common-objects').Request,
+  Request = require('kuzzle-common-objects').Request;
+
+// needs to be a variable so that unit tests can stub it
```

Comment:
Nitpicking: Make the source code test compliant makes me feel sad.
We could use mock-require instead of rewire (https://www.npmjs.com/package/mock-require); It would at least limit these kind of cases (same for other files in the same case)